hrs-portlets-webapp/src/main/webapp/WEB-INF/tags/hrs/pagerNavBar.tag

```diff
@@ -9,15 +9,15 @@
 <div class="flc-pager-${position} dl-pager-navbar" style="display:none">
   <ul class="dl-pager-bar fl-pager-ui uw-pager-bar">
     <li class="flc-pager-previous dl-pager-previous">
-      <a href="javascript:;" title="Previous Page" class="btn btn-default">Previous</a>
+      <a href="#" title="Previous Page" class="btn btn-default">Previous</a>
```

Comment:
Ok, I see. it has been called. should switch back.

On Thu, Aug 13, 2015 at 11:34 AM, Tim Levett <notifications@github.com>
wrote:

> In hrs-portlets-webapp/src/main/webapp/WEB-INF/tags/hrs/pagerNavBar.tag
> <https://github.com/UW-Madison-DoIT/hrs-portlets/pull/36#discussion_r36995189>
> :
>
> > @@ -9,15 +9,15 @@
> >  <div class="flc-pager-${position} dl-pager-navbar" style="display:none">
> >    <ul class="dl-pager-bar fl-pager-ui uw-pager-bar">
> >      <li class="flc-pager-previous dl-pager-previous">
> > -      <a href="javascript:;" title="Previous Page" class="btn btn-default">Previous</a>
> > +      <a href="#" title="Previous Page" class="btn btn-default">Previous</a>
>
> The reason it is javascript:; is because it runs some javascript code
> when you click it. this is by design. Where is it suggested that we change
> these to #? This may break things for fluid, was testing for non-screen
> readers done on this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Madison-DoIT/hrs-portlets/pull/36/files#r36995189>.
>